atlassian_jwt_auth/contrib/tests/django/test_django.py

```diff
@@ -0,0 +1,85 @@
+import django
+import os
+
+from django.test.testcases import SimpleTestCase
+from django.test.utils import override_settings
+
+from atlassian_jwt_auth import create_signer
+from atlassian_jwt_auth.contrib.tests.utils import get_static_retriever_class
+from atlassian_jwt_auth.tests import utils
+
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE',
+                      'atlassian_jwt_auth.contrib.tests.django.settings')
+
+django.setup()
+
+
+def create_token(issuer, audience, key_id, private_key):
+    signer = create_signer(issuer, key_id, private_key)
+
+    return signer.generate_jwt(audience)
+
+
+class TestAsapDecorator(SimpleTestCase):
+    _private_key_pem = utils.get_new_rsa_private_key_in_pem_format()
```

Comment:
I am requiring that new tests use the test mixins that are available so that you can test both RSA and ECDSA algorithms - please update the code accordingly.